internal/public/endpoints/context.go

```diff
@@ -10,9 +10,16 @@ import (
 	"github.com/jamespfennell/transiter/internal/public/reference"
 )
 
+type EndpointOptions struct {
+	// The maximum number of stops that can be returned in a single request.
+	// Defaults to 100.
```

Comment:
Sort of a nit, but the default exists on the flag not the struct, so I would omit the second line here.

Reply:
To clarify, do you think we should keep the `EndpointOptions` struct as optional and populate it with defaults (for now just this single option as 100) or just assume it will also be passed in and remove all defaults? Specifically referring to this code in `public.go`:

```go
if endpointOptions == nil {
    endpointOptions = &endpoints.EndpointOptions{MaxStopsPerRequest: 100}
}
```

Reply:
Ohhhh I see! Actually either way sounds good to me, but for sure if we keep the code as-is then the comment makes sense and we should keep it. Sorry for misunderstanding!

Reply:
Sounds good, will keep as-is.